sbin/pfdhcplistener

```diff
@@ -213,8 +213,7 @@ sub process_pkt {
             # For these reasons, we must peek directly at the data.
             # skip the first 12 bytes of the frame (src + dst eth addr)
             # grab the ether_type field and discard the rest:
-            my ($ether_type, undef) = unpack("x[12]nC*", $pkt);
-            if ($ether_type != NetPacket::Ethernet::ETH_TYPE_IP) {
+            if ( ($l2->{type} ne ETH_TYPE_IP) && ($l2->{type} ne ETH_TYPE_IPv6) ) {
```

Comment:
This check should be done earlier in the code.
Right after 
```
my $l2 = NetPacket::Ethernet->decode($pkt);
```
So we can exit as early as possible